packages/info/README.md

```diff
@@ -9,22 +9,55 @@ This package returns a set of information related to the local environment.
 ## Installation
 
 ```bash
+#npm
 npm i -D @webpack-cli/info
+
+#yarn
+yarn add @webpack-cli/info -D
 ```
 
 ## Usage
 
+### Args / Flags
+
+#### Output format
+
+| Flag         | Description                   | Type        |     |     |
+| ------------ | ----------------------------- | ----------- | --- | --- |
+| `--json`     | To get the output as JSON     | [ boolean ] |     |     |
+| `--markdown` | To get the output as markdown | [ boolean ] |     |     |
+
+#### Options
+
+| Flag                   | Description                                                    | Type        |
+| ---------------------- | -------------------------------------------------------------- | ----------- |
+| `--help`               | Show help                                                      | [ boolean ] |
+| `--version`            | Show version number                                            | [ boolean ] |
```

Comment:
Version number of what? We need to be specific in the description

Reply:
The version of `webpack-cli`
Just updated the docs :+1: